### PR TITLE
Add dummyserver v1 to Alire index

### PR DESCRIPTION
**Pull request by bracke**

### CODE REVIEW

File: index/du/dummyserver/dummyserver-1.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = "dummyserver"
+description = "DummyServer is a terminal program that serves dummy content"
+long-description = "DummyServer is a terminal program that serves dummy content (resources). These resources are defined in a single JSON configuration and configuration is thus very simple and fast. The prime purpose of DummyServer is to serve content to test client applications."
+licenses = "CC0-1.0"
+website = "https://github.com/bracke/dummyserver"
+tags = ["server", "test","terminal", "console"]
```
Comment (mosteo) [inline index/du/dummyserver/dummyserver-1.0.0.toml]: Since this seems to serve hypertext pages, maybe `http` would fit here besides just server?

**Review by mosteo — DISMISSED** [review]

File: index/du/dummyserver/dummyserver-1.0.0.toml (hunk 1 above)
Comment (bracke) [reply index/du/dummyserver/dummyserver-1.0.0.toml]: True :-)

Comment (bracke) [reply index/du/dummyserver/dummyserver-1.0.0.toml]: Do I need to do another pull request to change that?

Comment (mosteo) [reply index/du/dummyserver/dummyserver-1.0.0.toml]: No, you can just update the submitted manifest `dummyserver-1.0.0.toml`. The one in your repository can be updated for a future release to be in sync, but that doesn't merit updating the commit right now as the information Alire uses comes from the indexed manifest.

Comment (bracke) [reply index/du/dummyserver/dummyserver-1.0.0.toml]: Done.

**Review by mosteo — APPROVED** [review]

**Comment by mosteo** [comment]
Merged, thanks

**Pull request merged**